dglgo/dglgo/pipeline/graphpred/graphpred.jinja-py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from dgl.data import AsGraphPredDataset
+from dgl.dataloading import GraphDataLoader
+from sklearn.metrics import roc_auc_score
+from tqdm import tqdm
+{{ data_import_code }}
+
+{{ model_code }}
+
+def train(device, loader, model, criterion, optimizer):
+    model.train()
+
+    for _, (g, labels) in enumerate(tqdm(loader, desc="Iteration")):
+        if g.batch_size == 1:
```

Comment:
Why?

Reply:
It's necessary if the final prediction MLP consists of a batch normalization layer, which requires the batch size to be greater than 1 during training. This is not needed for this model, but might be needed for other models.

Reply:
Removed for now, might add it back if necessary